python/sidecar/settings.py

```diff
@@ -35,17 +39,46 @@ def make_settings_json_path(user_id: str) -> Path:
     return filepath
 
 
+def default_settings() -> FlatSettingsSchema:
+    return FlatSettingsSchema(
+        current_directory="",
+        logging_enabled=False,
+        logging_filepath="/tmp/refstudio-sidecar.log",
+        openai_api_key="",
+        openai_chat_model="gpt-3.5-turbo",
+        openai_manner=RewriteMannerType.SCHOLARLY,
+        openai_temperature=0.7,
+    )
+
+
 def initialize_settings_for_user(user_id: str) -> None:
     filepath = make_settings_json_path(user_id)
     filepath.parent.mkdir(parents=True, exist_ok=True)
 
-    defaults = SettingsSchema()
+    defaults = default_settings()
 
     with open(filepath, "w") as f:
         json.dump(defaults.dict(), f)
 
 
-def get_settings_for_user(user_id: str) -> SettingsSchema:
+def migrate_settings(old_settings: dict) -> FlatSettingsSchema:
```

Comment:
wondering how/when we will get rid of this migration (fallback) code.

Reply:
Maybe in a week? Or since we don't care about backwards compatibility yet, you could be bold and take it out now :)